pkg/opts/devicemappings.go

```diff
@@ -7,6 +7,25 @@ import (
 	"github.com/alibaba/pouch/apis/types"
 )
 
+// ParseDeviceMappings parse devicemappings
+func ParseDeviceMappings(devices []string) ([]*types.DeviceMapping, error) {
+	results := []*types.DeviceMapping{}
+	for _, device := range devices {
+		deviceMapping, err := ParseDevice(device)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse devices: %v", err)
+		}
+
+		if !ValidDeviceMode(deviceMapping.CgroupPermissions) {
+			return nil, fmt.Errorf("%s invalid device mode: %s", device, deviceMapping.CgroupPermissions)
+		}
+
+		results = append(results, deviceMapping)
+	}
+	return results, nil
+
+}
+
 // ParseDevice parses a device mapping string to a container.DeviceMapping struct
 func ParseDevice(device string) (*types.DeviceMapping, error) {
```

Comment:
If this a function which is only used in package opts, I suggest that we make it private with `parseDevice`.